### DISCUSSION TITLE
How does one replace values for a ChatPromptTemplate for Langchain

### DISCUSSION THREAD

**Discussion by ajram23** [langfuse/langfuse#2200] [Support]
get_langchain_prompt doesnt work once compile is used.  May be I am missing something.

**Comment by marcklingen** [comment]
Usually when using get_langchain_prompt, you'd then let Langchain fill in the variables into your prompt that you have pulled from Langfuse.

get_langchain_prompt transforms the prompt in order for it to be parsed by Langchain.

**Reply by ajram23** [reply]
Not all variables are filled through Langchain, I'd have to fill some based on user preferences. How does one do this?

**Reply by marcklingen** [reply]
Interesting, could you pass these through via Langchain as well?

**Reply by ajram23** [reply]
Yes I can.

**Comment by ajram23** [comment]
This has now been implemented! Closing!
